back/src/Domain/Entity/Post.php

```diff
@@ -3,7 +3,8 @@
 namespace App\Domain\Entity;
 use App\Domain\Model\PostContent;
 use App\Domain\Model\PostTitle;
-class Post
+
+class Post implements \JsonSerializable
 {
     public function __construct(
         private readonly PostTitle     $title,
```

Comment:
I think your code would be more readable by making this property public and removing the getter.